### DISCUSSION TITLE
Can we change the default path of a custom dataset?

### DISCUSSION THREAD

**Discussion by roleiland** [danielegrattarola/spektral#340] [Q&A]
I'm creating my own dataset, and I would like to input the file/folder that contains my data. However, I found out that I cannot set or overwrite the default path, because it is a property variable. Is there an elegant way to do it?

**Comment by danielegrattarola** [comment]
The correct way to do it is to override the `path` property:

```python
@property
    def path(self):
        return "whatever/path/you/want"
```

Cheers
